ui/src/app/richskill/import/batch-import.component.html

```diff
@@ -139,6 +139,9 @@ <h3 class="t-type-bodyLargeBold t-type-text1 t-margin-extraSmall t-margin-bottom
           (fieldMappingChanged)="handleMappingChanged($event)"
         ></app-field-mapping-table>
       </div>
+      <div class="l-container l-container-mobileMedium t-margin-medium t-margin-bottom">
+        <app-batch-import-destination-select (mappingChanged)="updateTarget($event)"></app-batch-import-destination-select>
```

Comment:
This should either:
A) Default to `RSD Library` as destination and not have a prompt value
B) Keep the existing options but throw an error/prevent progress if no selection is made.

Reply:
it has the RSD Library destination as default if no one is provided.